Index: /mnt/Share/PublicShare/Projects/pspdev/libpng/Makefile
===================================================================
--- /mnt/Share/PublicShare/Projects/pspdev/libpng/Makefile	(revision 1816)
+++ /mnt/Share/PublicShare/Projects/pspdev/libpng/Makefile	(working copy)
@@ -5,7 +5,7 @@
 	pngread.o pngrio.o pngrtran.o pngrutil.o pngset.o \
 	pngtrans.o pngwio.o pngwrite.o pngwtran.o pngwutil.o
 
-CFLAGS = -O2 -G0
+CFLAGS = -O2 -G0 -mno-explicit-relocs
 
 include $(PSPSDK)/lib/build.mak
 
